samples/basic: Update readme for blinky/button/disco

erwango opened the pull request:
blinky/button and disco samples are now using definitions
generated from  boards dts files.
Update smaples README files accordingly.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder commented on samples/basic/blinky/README.rst:
```diff
@@ -14,13 +14,13 @@ Requirements
 The demo assumes that an LED is connected to one of GPIO lines. The
-sample code is configured to work on boards with user defined buttons and that
-have defined the LED0\_* variables in :file:`board.h`.
+sample code is configured to work on boards with that have defined the led0
```
```suggestion
sample code is configured to work on boards that have defined the led0
```

dbkinder commented on samples/basic/blinky/README.rst:
```diff
@@ -14,13 +14,13 @@ Requirements
-sample code is configured to work on boards with user defined buttons and that
-have defined the LED0\_* variables in :file:`board.h`.
+sample code is configured to work on boards with that have defined the led0
+alias in board device tree description file.
```
```suggestion
alias in their board device tree description file.
```

dbkinder commented on samples/basic/blinky/README.rst:
```diff
@@ -14,13 +14,13 @@ Requirements
+alias in board device tree description file.
 
-The :file:`board.h` must define the following variables:
+Doing so, following variables will be generated:
```
delete blank line 19 to put the sentence on line 20 in the same paragraph as the previous sentence.

dbkinder replied:
```suggestion
Doing so will generate these variables:
```

dbkinder commented on samples/basic/blinky/README.rst:
```diff
@@ -14,13 +14,13 @@ Requirements
 
-- LED0_GPIO_NAME (or LED0_GPIO_CONTROLLER)
-- LED0_GPIO_PIN
+ - LED0_GPIO_CONTROLLER
```
Delete the leading spaces on these two lines:
```suggestion
- LED0_GPIO_CONTROLLER
```

dbkinder commented on samples/basic/blinky/README.rst:
```diff
@@ -14,13 +14,13 @@ Requirements
-- LED0_GPIO_NAME (or LED0_GPIO_CONTROLLER)
-- LED0_GPIO_PIN
+ - LED0_GPIO_CONTROLLER
+ - LED0_GPIO_PIN
```
```suggestion
- LED0_GPIO_PIN
```

dbkinder commented on samples/basic/button/README.rst:
```diff
@@ -13,26 +13,16 @@ Requirements
-- :ref:`quark_d2000_devboard`
-- quark_se_c1000_devboard
-- quark_se_c1000_ss_devboard
+Alternatively, this could also be done by defining 'sw0' alias in board device
```
```suggestion
 Alternatively, this could also be done by defining 'sw0' alias in the board device
```

dbkinder commented on samples/basic/disco/README.rst:
```diff
@@ -47,27 +33,3 @@ The sample can be found here: :file:`samples/basic/disco`.
 
 Nucleo F103RB
```
Should this heading be deleted too?

erwango commented:
@dbkinder , Txs for the review, all comments addressed

erwango committed:
samples/basic: Update readme for blinky/button/disco

blinky/button and disco samples are now using definitions
generated from  boards dts files.
Update samples README files accordingly.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder reviewed (approved):
LGTM, thanks.
